tests/test_process/test_extended_frame_helper.py

```diff
@@ -0,0 +1,85 @@
+import cv2
+import yaml
+import numpy as np
+import pytest
+
+from mio.models.process import DenoiseConfig, NoisePatchConfig
+from mio.process.frame_helper import NoiseDetectionHelper
+
+from ..conftest import DATA_DIR
+
+
+@pytest.mark.parametrize(
+    "noise_detection_method",
+    [
+        "gradient",  # Only testing the "gradient" method
+    ],
+)
+def test_noise_detection_contrast_with_categories(noise_detection_method):
```

Comment:
ya ya as above, i think this should either replace the existing test function, or else we should add video/ground truth as a parameter and run this test over several different videos. no reason to duplicate code here

Reply:
I'll do this rn

Reply:
done in https://github.com/Aharoni-Lab/mio/pull/97/commits/3fdb6e44840b3c22e6274ec0e726fe4e6eb062bb